### PR TITLE
ActiveMQ_package(#606)

### PR DESCRIPTION
**Pull request by kush-crest**
## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [x] Each data streams should contains system tests.
- [x] Each data streams should contains sample_event.json.
- [x] Each data streams should successfully pass all tests.

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/activemq directory.
- Run following command to run system tests. elastic-package test system -v

## Related issues

- Relates #606

### CODE REVIEW

File: packages/activemq/data_stream/broker/fields/ecs.yml [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+    - name: address
+      level: core
+      type: keyword
+      ignore_above: 1024
+      description: 'The type of the service data is collected from.
+
+        The type can be used to group and correlate logs and metrics from one service type.
+
+        Example: If logs or metrics are collected from Elasticsearch, `service.type` would be `elasticsearch`.'
+      example: elasticsearch
```
Comment (jsoriano) [inline packages/activemq/data_stream/broker/fields/ecs.yml:375]: Please use `external` notation for ECS fields as done here for example: https://github.com/elastic/integrations/blob/227e245005f08881ffcb9535b393e08aa363c2e0/packages/apache/data_stream/access/fields/ecs.yml
This way you don't need to copy all the descriptions and so on.

File: packages/activemq/data_stream/queue/fields/fields.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+- name: activemq.queue
+  type: group
+  release: ga
```
Comment (jsoriano) [inline packages/activemq/data_stream/queue/fields/fields.yml:3]:
```suggestion
  release: beta
```

File: packages/activemq/data_stream/queue/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+type: metrics
+title: ActiveMQ queue metrics
+release: experimental
```
Comment (jsoriano) [inline packages/activemq/data_stream/queue/manifest.yml:3]:
```suggestion
release: beta
```

File: packages/activemq/data_stream/topic/fields/fields.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+- name: activemq.topic
+  type: group
+  release: ga
```
Comment (jsoriano) [inline packages/activemq/data_stream/topic/fields/fields.yml:3]:
```suggestion
  release: beta
```

File: packages/activemq/data_stream/topic/manifest.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+type: metrics
+title: ActiveMQ topic metrics
+release: experimental
```
Comment (jsoriano) [inline packages/activemq/data_stream/topic/manifest.yml:3]:
```suggestion
release: beta
```

File: packages/activemq/data_stream/audit/manifest.yml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+type: logs
+title: ActiveMQ audit logs
+release: experimental
```
Comment (jsoriano) [inline packages/activemq/data_stream/audit/manifest.yml:3]:
```suggestion
release: beta
```

File: packages/activemq/data_stream/broker/fields/fields.yml [hunk 7]
```diff
@@ -0,0 +1,40 @@
+- name: activemq.broker
+  type: group
+  release: ga
```
Comment (jsoriano) [inline packages/activemq/data_stream/broker/fields/fields.yml:3]: Let's start with beta.
```suggestion
  release: beta
```

File: packages/activemq/data_stream/broker/manifest.yml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+type: metrics
+title: ActiveMQ broker metrics
+release: experimental
```
Comment (jsoriano) [inline packages/activemq/data_stream/broker/manifest.yml:3]: Experimental should be used only for features that we don't plan to give any kind of support in the long term in their current form.
```suggestion
release: beta
```

File: packages/activemq/data_stream/log/manifest.yml [hunk 9]
```diff
@@ -0,0 +1,17 @@
+type: logs
+title: ActiveMQ log logs
+release: experimental
```
Comment (jsoriano) [inline packages/activemq/data_stream/log/manifest.yml:3]:
```suggestion
release: beta
```

File: packages/activemq/data_stream/log/fields/ecs.yml [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+  external: ecs
+- name: log.level
+  external: ecs
+- name: log.file.path
+  external: ecs
+- name: ecs.version
+  external: ecs
+- name: log.flags
+  type: keyword
+  description: Log flags
```
Comment (jsoriano) [inline packages/activemq/data_stream/log/fields/ecs.yml:11]:
```suggestion
- name: log.flags
  external: ecs
```

File: packages/activemq/data_stream/queue/fields/ecs.yml [hunk 11]
```diff
@@ -0,0 +1,375 @@
...
+    - name: address
+      level: core
+      type: keyword
+      ignore_above: 1024
+      description: 'The type of the service data is collected from.
+
+        The type can be used to group and correlate logs and metrics from one service type.
+
+        Example: If logs or metrics are collected from Elasticsearch, `service.type` would be `elasticsearch`.'
+      example: elasticsearch
```
Comment (jsoriano) [inline packages/activemq/data_stream/queue/fields/ecs.yml:375]: Use `external` for ECS fields.

File: packages/activemq/data_stream/topic/fields/ecs.yml [hunk 12]
```diff
@@ -0,0 +1,375 @@
...
+    - name: address
+      level: core
+      type: keyword
+      ignore_above: 1024
+      description: 'The type of the service data is collected from.
+
+        The type can be used to group and correlate logs and metrics from one service type.
+
+        Example: If logs or metrics are collected from Elasticsearch, `service.type` would be `elasticsearch`.'
+      example: elasticsearch
```
Comment (jsoriano) [inline packages/activemq/data_stream/topic/fields/ecs.yml:375]: Use `external`.

File: packages/activemq/manifest.yml [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+  - src: /img/activemq.svg
+    title: activemq
+    size: 32x32
+    type: image/svg+xml
+format_version: 1.0.0
+license: basic
+categories:
+  - web
+conditions:
+  kibana.version: ^7.9.0
```
Comment (jsoriano) [inline packages/activemq/manifest.yml:17]: We may want to use `7.14.1` or `7.15.0` here.

**Review by jsoriano — CHANGES_REQUESTED** [review]

File: packages/activemq/manifest.yml (hunk 13 above)
Comment (kush-crest) [reply packages/activemq/manifest.yml:17]: packages/activemq/manifest.yml
```version: 0.0.1
release: experimental
description: ActiveMQ Integration
```

Do i need to change release: experimental to release: beta here too?

File: packages/activemq/_dev/deploy/docker/Dockerfile [hunk 14]
```diff
@@ -0,0 +1,39 @@
+FROM openjdk:8-jre-alpine
+
+ENV ACTIVEMQ_VERSION=5.15.9
```
[the remark this answers is not in the document]
Comment (kush-crest) [reply packages/activemq/_dev/deploy/docker/Dockerfile]: Done

File: packages/activemq/data_stream/audit/agent/stream/log.yml.hbs [hunk 15]
```diff
@@ -0,0 +1,10 @@
+paths:
+{{#each paths as |path i|}}
+ - {{path}}
+{{/each}}
+exclude_files: [".gz$"]
+processors:
+  - add_fields:
+      target: ''
+      fields:
+        ecs.version: 1.10.0
```
[the remark this answers is not in the document]
Comment (kush-crest) [reply packages/activemq/data_stream/audit/agent/stream/log.yml.hbs]: Done

File: packages/activemq/data_stream/broker/fields/ecs.yml (hunk 1 above)
Comment (kush-crest) [reply packages/activemq/data_stream/broker/fields/ecs.yml]: Done

File: packages/activemq/data_stream/queue/fields/fields.yml (hunk 2 above)
Comment (kush-crest) [reply packages/activemq/data_stream/queue/fields/fields.yml]: Done

File: packages/activemq/data_stream/queue/manifest.yml (hunk 3 above)
Comment (kush-crest) [reply packages/activemq/data_stream/queue/manifest.yml]: Done

File: packages/activemq/data_stream/topic/fields/fields.yml (hunk 4 above)
Comment (kush-crest) [reply packages/activemq/data_stream/topic/fields/fields.yml]: Done

File: packages/activemq/data_stream/topic/manifest.yml (hunk 5 above)
Comment (kush-crest) [reply packages/activemq/data_stream/topic/manifest.yml]: Done

File: packages/activemq/data_stream/audit/manifest.yml (hunk 6 above)
Comment (kush-crest) [reply packages/activemq/data_stream/audit/manifest.yml]: Done

File: packages/activemq/data_stream/broker/fields/fields.yml (hunk 7 above)
Comment (kush-crest) [reply packages/activemq/data_stream/broker/fields/fields.yml]: Done

File: packages/activemq/data_stream/broker/manifest.yml (hunk 8 above)
Comment (kush-crest) [reply packages/activemq/data_stream/broker/manifest.yml]: Done

File: packages/activemq/data_stream/log/manifest.yml (hunk 9 above)
Comment (kush-crest) [reply packages/activemq/data_stream/log/manifest.yml]: Done

File: packages/activemq/data_stream/log/fields/ecs.yml (hunk 10 above)
Comment (kush-crest) [reply packages/activemq/data_stream/log/fields/ecs.yml]: Done

File: packages/activemq/data_stream/queue/fields/ecs.yml (hunk 11 above)
Comment (kush-crest) [reply packages/activemq/data_stream/queue/fields/ecs.yml]: Done

File: packages/activemq/data_stream/topic/fields/ecs.yml (hunk 12 above)
Comment (kush-crest) [reply packages/activemq/data_stream/topic/fields/ecs.yml]: Done

File: packages/activemq/manifest.yml (hunk 13 above)
Comment (kush-crest) [reply packages/activemq/manifest.yml]: Done

**Pull request merged**